mne_bids/tests/test_path.py

```diff
@@ -166,6 +168,118 @@ def test_get_entity_vals(entity, expected_vals, kwargs, return_bids_test_dir):
     shutil.rmtree(deriv_path)
 
 
+def test_path_benchmark(tmp_path_factory):
+    """Benchmark exploring bids tree."""
+    tmp_bids_root = tmp_path_factory.mktemp("abc")
+
+    derivatives = [Path("derivatives", "derivatives" + str(i)) for i in range(17)]
+
+    bids_subdirectories = ["", "sourcedata", *derivatives]
+
+    def equal_length_subj_ids(v):
+        assert all(v > 0)
+        n_digits = len(str(np.max(v))) + 1
+        return list(map(lambda i: "0" * (n_digits - len(str(i))) + str(i), v))
+
+    # Create a BIDS compliant directory tree with high number of branches
+    for i in equal_length_subj_ids(np.arange(1, 20, dtype=int)):
+        for j in range(1, 9):
```

Comment:
where are the 20 and 9 coming from? as with the 17 above, better define these as variables within the test, potentially with a quick inline comment, or even better, a descriptive var name

```
n_test_subs = 20
...
```